common/components/UI/RealEstateComponents/RealEstateModal/RealEstateForm/index.tsx

```diff
@@ -20,6 +21,17 @@ const RealEstateForm: FC<Props> = ({
   editable = true,
   setIsValueChanged,
 }) => {
+  const { data: session } = useSession()
+  const [initialValue, setInitial] = useState<{ [key: string]: any }>({
+    adminEmails: [],
+  })
+  useEffect(() => {
+    if(session?.user.email){
+      setInitial({
+        adminEmails: [session?.user.email],
+      })
+    }
+  }, [form])
```

Comment:
Same as above (comment to `common/components/UI/DomainsComponents/DomainModal/DomainForm/index.tsx`)